lib/shared/permissions.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 resin.io
```

Comment:
Something else to _maybe_ update, when we decide what we're doing with the Copyright years ;)